galaxyproject/galaxy
# UI, histories: fix scroll position capture/restore for the current hi…

…story

To test:
- Using a longer history, expand and scroll down to the history's lower edge
- Go to the workflows page using the masthead link
- Return to the analyze page using the masthead link
- The history should be at the scroll position you left it at
